cli/src/commands/init.js

```diff
@@ -160,7 +160,23 @@ const handler = async ({ force, name, cwd, lib, typeScript }) => {
         reporter.info('Installing @dhis2/cli-app-scripts...')
         await exec({
```

Comment:
react and react-dom are peer dependencies for most of our toolset - this worked before coincindentally because React was hoisted from app-shell, but apps should have the dependency explicit.